src/authorization/AuxiliaryAuthorizer.ts

```diff
@@ -0,0 +1,55 @@
+import type { AuxiliaryManager } from '../ldp/auxiliary/AuxiliaryManager';
+import { getLoggerFor } from '../logging/LogUtil';
+import { NotImplementedHttpError } from '../util/errors/NotImplementedHttpError';
+import type { AuthorizerArgs } from './Authorizer';
+import { Authorizer } from './Authorizer';
+
+/**
+ * An authorizer for auxiliary resources.
+ * Calls the source authorizer with potentially updated permissions
```

Comment:
Perhaps add some explanation of how permissions for auxiliary resources actually depend on those on other resource, otherwise this is hard to understand.